AUTHORS

```diff
@@ -0,0 +1,11 @@
+Richard Gowers
```

Comment:
Maybe worth adding context here? - indeed the upstream openfe library needs its own AUTHORS & context (not had time to open that issue yet).

Something along the lines of "includes vendored files from X", probably also needs a "OpenMM protocols derive from work in Perses and Yank".